custom_components/opnsense/coordinator.py

```
@@ -71,225 +72,231 @@
+                            (
+                                new_property,
+                                value,
+                            ) = await OPNsenseDataUpdateCoordinator._calculate_speed(
+                                prop_name=prop_name,
+                                elapsed_time=elapsed_time,
+                                current_parent_value=interface[prop_name],
+                                previous_parent_value=previous_interface[prop_name],
+                            )
 
-                    interface[new_property] = value
+                        interface[new_property] = value
```

Pylint (3.3.7)

[error] Possibly using variable 'new_property' before assignment

(E0606)